### DISCUSSION TITLE
3D Scans in dipy

### DISCUSSION THREAD

**Discussion by Little-Foot-Shapes** [dipy/dipy#2301] [3. Community Lounge]
Hello, 

I was wondering if this software can work with 3D scans of feet? I have 148 foot scans from different age groups and 3D shape data for each point on the foot. I want to do SPM on this data. I have the data in stl or obj or other usual 3D scan format or in point cloud in MATLAB.

Thank you.

Best Regards
Mat

**Comment by StongeEtienne** [comment]
Hi, Dipy is a specialized tool for **Diffusion** MRI,
If you goal is to visualise 3D data points (point cloud), I higly suggest you to look into VTK : https://vtk.org/  
or maybe Fury: https://github.com/fury-gl/fury

I'm not exacly sure for the SPM analysis however, since its normaly for **functionnal** MRI.
Those statistical analysis can be done in Python, but Dipy does not include those.
